src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,17 +2,19 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int NUM_SIZE = 12;
+    public static final int NO_NUMS = -1;
+    private int[] numbers = new int[NUM_SIZE];
```

Comment:
You can add final here, sine you don't ever change the address of this variable.